engines/bastion/app/assets/javascripts/bastion/components/nutupane.factory.js

```diff
@@ -118,8 +126,9 @@ angular.module('Bastion.components').factory('Nutupane',
 
                     table.rows = response.results;
 
-                    table.resource.page = parseInt(response.page, 10);
-                    params.page = table.params.page = parseInt(response.page, 10);
+                    responsePage = response.page || 1;
```

Comment:
This will return `undefined` if the response does not have a page number, which it doesn't in a response for a single object. For example, for CV Filter Rules, we GET a single CV filter and then iterate through the `rules` attribute returned. There is no paging information in this response. This returns undefined and sets the page as undefined.

It has been working fine because the CV filter rules page is not paginated, but adding paginated pages to the same page exposed the issue.